src/setupWorker/glossary.ts

```diff
@@ -56,6 +56,21 @@ export interface SetupWorkerInternalContext {
   requestHandlers: RequestHandler<any, any>[]
   emitter: StrictEventEmitter
   keepAliveInterval?: number
+  workerChannel: {
```

Comment:
I'm scoping the way to send and receive events from the worker to these typed methods, so that unknown events cannot be sent/listened to.